research/gnn-survey-paper/layers.py

```diff
@@ -0,0 +1,29 @@
+import tensorflow as tf
+
+class GraphConvLayer(tf.keras.layers.Layer):
+
```

Comment:
Here you shouldn't use a dictionary. It's better to have an input_dim, output_dim, and bias as constructor arguments.

You might want to pass in **kwargs and forward it to __init__()

Reply:
OK! I've already converted it to pass in **kwargs~
Thanks~~